### PR TITLE
Train accuracy

### PR DESCRIPTION
**Pull request by svebk**
Issue was with label_batch shape. Using `label_batch = tf.squeeze(label_batch)` fixes this.

### CODE REVIEW

File: deepmodels/tf/core/learners/dm_classifier.py [hunk 1]
```diff
@@ -26,9 +24,20 @@ def comp_train_accuracy(pred_logits, label_batch):
...
     top 1 classfication accuracy.
   """
-  correct_prediction = tf.equal(tf.argmax(pred_logits, 1), label_batch)
-  accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
-  tf.scalar_summary("eval/mean_clf_accuracy", accuracy)
+
+  # issue with img_folder_data that produces 2 dimensional label_batch
+  # squeeze fix that
+  label_batch = tf.squeeze(label_batch)
+  pred_cls = tf.cast(tf.argmax(pred_logits, 1), tf.int32)
```
Comment (flyfj) [inline deepmodels/tf/core/learners/dm_classifier.py]: these type checks seem to be necessary during computing, we may need to make a more rigorous check at other places too.
